### ISSUE TITLE
[NEW] Minimal preferences pane

### ISSUE THREAD

**Issue by ghost** [RocketChat/Rocket.Chat.Android#186]
Your Rocket.Chat.Android version: (I'm pretty sure running the latest;-)

My crew asked for some options to adjust the mobile app. Not all options we have at the site for now, but they would like to change the avatar for example.

Of course, I see this as a non priority thing right now - just to keep in mind for a future release!

**Comment by laggedHero** [comment]
Hello @jm4rc05.

Yes, we are missing preferences in app. Would you like to help us define what a "minimal" would be?

You have the avatar example. Any other your crew is missing?

Thank you.

**Comment by ghost** [comment]
Hi @laggedHero , since you asked ;-), lets put it in importance order and risking to see something
 as new features or issues!

Essential:
1. Notifications (on/off - i.e. enable/disable notifications):
    - Notification sound (on/off) - could use whatever the system have for the sound itself and in the future you can be able to customize;
    - Vibrate (on/off);
2. Media:
    - Download by default or do not download (related to [#135](https://github.com/RocketChat/Rocket.Chat.Android/issues/135) and [#181](https://github.com/RocketChat/Rocket.Chat.Android/issues/181) perhaps);
3. Profile:
    - Change password;

Would be good to have:
1. Profile:
    - Change name;
    - Change avatar (they agree that this is not so important after all ;-)

**Comment by laggedHero** [comment]
Hello @jm4rc05.

Nice list. The `media download by default or do not` relates to a setting that already exists on the web and we are aware of it. :)

@geekgonecrazy @graywolf336 @sampaiodiego Would you like to propose something else here?

**Comment by geekgonecrazy** [comment]
Biggest two to me are on the list already but are:
* Notification sound  
* Allow toggeling of "media download by default" setting which relates to: #192

Others would be useful, but nothing else do I really feel it would be too big of a hassle to open browser on phone if I need to.:smile:

**Comment by philipbrito** [comment]
Hey guys, I'm just updating the status of this issue. 

Currently it is already possible to  change the `name`, `username`, `email`, `password` and `avatar (by using an URL)`.

In the next version we'll ship the possibility to `change the avatar by choosing an image from the gallery or by taking a photo directly from the app`, so stay tuned :)

Regarding `media` we are showing it on the message list and when opened (i.e the user taps on an image to see it in fullscreen) there is an option to download it to your device.

For sure we'll have the preferences on the app regarding `notifications`.

Thank you all.

**Comment by nemhods** [comment]
The ability to mute individual channels would be a huge feature for our installation, where we have our users auto-added to about 20 different channels. We get a lot of complaints (and some users even uninstalled the app) because users are getting spammed at an unfortunate time when they are not in reach of a computer to mute a channel through the desktop app.

**Comment by rafaelks** [comment]
@nemhods Can you specify what kind of "mute" are you talking about?

**Comment by nemhods** [comment]
@rafaelks of course! I'm talking about the "Notification Preferences" that are already available through the desktop app. 
We migrated from various custom WhatsApp and Facebook Groups, and Users are used to the workflow of muting an individual group via two or three taps from the context menu of the group.
![grafik](https://user-images.githubusercontent.com/6408566/47566002-9404da80-d92a-11e8-8ac1-cff0d7909d70.png)

**Comment by rafaelks** [comment]
@nemhods Gotcha! Yes, you can't change the notification preferences yet, we'll be working on that in the future for sure, thanks for letting us know that you care about it! 👍

**Comment by nemhods** [comment]
Thank you for looking after the issue :)
As per [this issue ](https://github.com/RocketChat/Rocket.Chat.iOS/issues/2096) I assume the feature is already in the iOS app? I don't have an iOS device to confirm currently.
If I am incorrect, someone please correct me so I can open an issue over there as well.

**Comment by rafaelks** [comment]
@nemhods Yes, iOS already have this option.

**Comment by rocketkittens** [comment]
Requesting the ability to change **Notification** settings found in My **Account -> Preferences**. This feature is missing from the android app, it is present in the Dekstop and Web versions.

Using v3.2.0
